cypress/integration/components/card-login.spec.js

```diff
@@ -0,0 +1,42 @@
+
+describe("Testa comportamento do componente CardLogin", () => {
+    it("Retorna true se o botão que dispara o componente está visível e clica no botão", () => {
+
+        cy.visit('/')
+
+        cy.get('.style_container__jFrh2').should('be.visible')
+        cy.get('.style_container__jFrh2').click({ multiple: true }).wait(500)
+    })
```

Comment:
dá para encadear as funções do cypress, verifica se está visível e clica
```suggestion
        cy.get('.style_container__jFrh2').should('be.visible') .click()

```